charts/nm-vllm/examples/basic/README.md

```diff
@@ -0,0 +1,59 @@
+# Basic
+
+This example demonstrates the values required to use the `nm-vllm` chart to
+deploy the `TheBloke/Mistral-7B-Instruct-v0.2-GPTQ` model.
+
+Because the target model is quantized using GPTQ, the values also specify extra
+arguments for the vLLM OpenAI API server to inform the server to use `float16`
+for the data type of the model weights and activations.
+
+Finally, the values configure the resource limits for the API server deployment
+such that the pod is limited to:
+
+- `4 cpus`
+- `16Gi memory`
+- `1 nvidia.com/gpu`
+
+## Usage
+
+### Using Neural Magic Helm Repository
+
+```bash
+# Add Neural Magic Helm repository
+helm repo add neuralmagic https://helm.neuralmagic.com
+
+# Install nm-vllm chart using basic example values
+helm install nm-vllm nm-vllm -f https://raw.githubusercontent.com/neuralmagic/helm-charts/main/charts/nm-vllm/examples/basic/values.yaml
+```
+
+### Using `helm-charts` git repository
+
+```bash
+# Clone Neural Magic helm-charts git repository
+git clone https://github.com/neuralmagic/helm-charts.git
+
+# Install nm-vllm chart using basic example values
+cd helm-charts
+helm install nm-vllm charts/nm-vllm -f charts/nm-vllm/examples/basic/values.yaml
+```
+
+## Values
+
+The content of `values.yaml` is included in full below, but the `values.yaml`
+file is also available [here](./values.yaml).
+
+```yaml
+apiServer:
+  extraArgs:
+    - --dtype
+    - float16
+
+modelName:
+  TheBloke/Mistral-7B-Instruct-v0.2-GPTQ
+
+resources:
+  limits:
+    cpu: 4
+    memory: 16Gi
+    nvidia.com/gpu: 1
```

Comment:
Is there a way to specify the GPU type?

e.g. A100 vs A10?

Reply:
I didn't think there was a general way to do this because the way we do it at NM isn't very general, but it seems like there is a general way and we use a custom way we deem to be simpler. I added a commit with some `nodeSelector` config here: https://github.com/neuralmagic/helm-charts/pull/8/commits/2b316d8e274845d46d9b62ded0349dae180d8f8e